src/badguy/willowisp.cpp

```diff
@@ -105,12 +156,31 @@ WillOWisp::after_editor_set()
                                  m_color.green * 0.2f,
                                  m_color.blue * 0.2f));
   m_sprite->set_color(m_color);
+
+  if (Editor::is_active() && has_valid_path())
+  {
+    const auto& nodes = get_path()->get_nodes();
+    const int size = static_cast<int>(nodes.size());
+    if (!nodes.empty())
+    {
+      if (m_starting_node < 0)
+      {
+        m_starting_node = 0;
+      }
+      if (m_starting_node >= size)
+      {
+        m_starting_node = size - 1;
+      }
+      set_pos(m_path_handle.get_pos(m_col.m_bbox.get_size(), nodes[m_starting_node].position));
```

Comment:
Isn't this code semantically identical to https://github.com/SuperTux/supertux/pull/3265/files#diff-3f748aafc72f1aa095eed38ee035f379c712ab01c3f0c7352ca355b16f12d279R129-R147 ?